components/applications-service/integration_test/services_test.go

```diff
@@ -486,6 +488,27 @@ func assertServicesEqual(t *testing.T, expected, actual []*applications.Service)
 				svc.Site,
 				actual[i].Site,
 				"The site of a service is not the expected one")
+
+			// By default the previous health check should be set to NONE
+			assert.Equal(t,
+				applications.HealthStatus_NONE,
+				actual[i].PreviousHealthCheck,
+				"The previous health check of a service is not the expected one")
+
+			// The health check should have been updated less than two seconds ago
+			healthUpdatedAt, err := ptypes.Timestamp(actual[i].HealthUpdatedAt)
+			assert.Nil(t, err)
+			//@afiune upgrate testify go package to be able to use 'assert.Lessf()'
```

Comment:
The version we're using has `InDelta` which is a little nicer https://godoc.org/github.com/stretchr/testify/assert#InDelta

Reply:
changed